client/services/tutors.js

```diff
@@ -1,2 +1,66 @@
-import axios from "axios"
+import api from "./apiConfig"
+
+export const getTutors = async () => {
+  const response = await api.get("/tutors")
+  const tutors = response.data;
+  return tutors;
+};
```

Comment:
might be a good idea to have try catch statement

Reply:
done